jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -141,6 +151,14 @@ properties:
             fits_keyword: TIME-OBS
             blend_table: True
             blend_rule: first
+          date_beg:
+            anyOf:
+              - $ref: http://stsci.edu/schemas/asdf/time/time-1.0.0
```

Comment:
Did you intend to use the "$ref" for date_end as well?

Reply:
I guess we could use that for DATE-END as well, because it is also in the format of a datetime string (like the DATE keyword, which also uses the time schema reference).